mods/bones/init.lua

```diff
@@ -218,12 +240,33 @@ minetest.register_on_dieplayer(function(player)
 			player_inv:set_list(list_name, {})
 		end
 		drop(pos, ItemStack("bones:bones"))
+
+		if enable_bones_logging then
+			minetest.log("action", "[bones] " .. player_name ..
+				"'s bones do not remain since cannot place bones here -- " ..
```

Comment:
Suggestion: "player_name died at pos, but bones do not appear because bones cannot be placed here."

Reply:
I really dislike "bones do not remain".  It comes across as poor english to me, instead of the way you intend it, and I don't see it adding to immersion. Maybe someone else can weigh in with their thoughts.

Reply:
'since / because cannot place bones here' is bad.

Reply:
`(player) died at (pos). Items are dropped because bones cannot be placed here`